lib/pdksync.rb

```diff
@@ -6,6 +6,7 @@
 require 'pdk'
 require 'pdksync/configuration'
 require 'pdksync/gitplatformclient'
+require 'pdksync/jenkinclient'
```

Comment:
Is this a typo?  should it be jenkinsclient?

I also don't feel we should tie pdksync to one specific CI system.  Additionally this library should only be used when it is required.  (lazy loaded).

Reply:
I've renamed it to jenkinsclient.